sbncode/CAFMaker/FillExposure.cxx

```diff
@@ -0,0 +1,39 @@
+
+#include "FillExposure.h"
```

Comment:
```suggestion
#include "sbncode/CAFMaker/FillExposure.h"
```

Reply:
Idle comment about this: I would argue that `FillExposure.h` and `FillExposure.cxx` living in the same directory is a requirement. If that's true, isn't actually a better choice to go Bruce's way?

Currently, we use to officially recommend Chris's approach anyway.

Reply:
I'm not sure what Bruce is doing, but I just followed the convention I see with all the other "Fill" files. I agree Chris's approach seems fine too.

Reply:
All of our code uses full paths, except for a few places that don't (lots of places in StandardRecord :cry:). If I was in charge I would require full paths everywhere. Been bitten too many times by things being picked up from unexpected places.